apps/wing-console/console/ui/src/services/use-tests.ts

```diff
@@ -26,10 +28,8 @@ export const useTests = () => {
   });
 
   useEffect(() => {
-    if (!testListQuery.data) {
-      return;
-    }
-    setTestList(testListQuery.data);
+    setTestList(testListQuery.data || []);
+    setTestsExists(!!testListQuery.data && testListQuery.data.length > 0);
   }, [testListQuery.data]);
```

Comment:
I'd rather keep the early return but use two different effects. It's more readable to me.